gbpservice/neutron/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -0,0 +1,130 @@
+import json
+import subprocess
+
+from neutron.common import rpc as n_rpc
+from neutron.agent.common import config
+from oslo_config import cfg
+from oslo_log import log as logging
+import oslo_messaging
+from pecan import expose, request, response
+from pecan import rest
+
+
+import constants
+
+LOG = logging.getLogger(__name__)
+
+"""Controller class for handling all the HTTP requests.
+This Controller class serves all the HTTP requests sent by
+config-agent to configurator and make RPC call accordingly."""
+
+
+class Controller(rest.RestController):
+
+    def __init__(self, method_name):
+        try:
+            self.host = subprocess.check_output(
+                            'hostname', shell=True).rstrip()
```

Comment:
Add all the instructions in the constructor to try block and change the log message to more generic.